modules/ops_manager/ops_manager.tf

```diff
@@ -118,21 +123,24 @@ resource "azurerm_network_interface" "ops_manager_nic" {
 
 resource "azurerm_virtual_machine" "ops_manager_vm" {
   name                          = "${var.env_name}-ops-manager-vm"
-  depends_on                    = ["azurerm_network_interface.ops_manager_nic", "azurerm_storage_blob.ops_manager_image"]
+  depends_on                    = ["azurerm_network_interface.ops_manager_nic"]
   location                      = "${var.location}"
   resource_group_name           = "${var.resource_group_name}"
   network_interface_ids         = ["${azurerm_network_interface.ops_manager_nic.id}"]
   vm_size                       = "${var.ops_manager_vm_size}"
   delete_os_disk_on_termination = "true"
 
+storage_image_reference {
```

Comment:
@drhpivotal Thank you for your contribution, just a minor comment: can you fix the indentation here?